ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java

```diff
@@ -112,6 +112,48 @@
    */
   CompletableFuture<Void> terminatePushQuery(String queryId);
 
+  /**
+   * Sends a SQL request to the ksqlDB server. This method supports 'CREATE', 'CREATE ... AS
+   * SELECT', 'DROP', 'TERMINATE', and 'INSERT INTO ... AS SELECT' statements.
```

Comment:
I considered introducing separate methods specific for terminating persistent queries and dropping streams/tables, but decided the additional complexity wasn't worth the minor benefit. Happy to hear other opinions on this, though.